script.js

```diff
@@ -1,4 +1,141 @@
+// BEAT THAT
+// Game mode (mode 1. play dice rolls) (mode 2. decide the dice order) (mode 3. decide the winner)
+// there are two players, two dices
+
+// global variables
+// randomly selected dice for each players
+var player1Dice1 = "";
+var player1Dice2 = "";
+var player2Dice1 = "";
+var player2Dice2 = "";
+
+// the numbers that decided by players
+var player1Num = "";
+var player2Num = "";
+
+//player score
+var player1Score = [];
+var player2Score = [];
+
+// game mode starts with player 1
+var gameMode = "player 1";
+
+// To get random number from dice and cover it to string
+var diceRoll = function () {
+  var randomNum = Math.floor(Math.random() * 6 + 1);
+  return randomNum.toString();
+};
+
+//Player 1 plays diceRoll
+var Player1DiceRoll = function () {
+  player1Dice1 = diceRoll();
+  player1Dice2 = diceRoll();
+  gameMode = "player 1 position";
+  return `Hello Player 1 <br><br> You rolled ( ${player1Dice1} ) for 🎲 one and ( ${player1Dice2} ) for 🎲 two. <br><br> Please choose the order of your dice 🎲 by entering 1 or 2. `;
+};
+
+//Player 1 choose dice number order
+var player1Choice = function (input) {
+  if (input == 1 || input == 2) {
+    player1Num = combineDiceRolls(input);
+    player1Score.push(player1Num);
+    gameMode = "player 2";
+    return `Player 1<br><br> Your chosen number is ( ${player1Num} ). <br><br> It is now Player 2's turn. <br><br> Please click 'Submit' to roll the dice `;
+  } else {
+    return verifyInput(input);
```

Comment:
[Comment] Very nice to have a verification function that will auto place message to user. Nice!